gradle/dependencies.gradle

```diff
@@ -237,14 +237,14 @@ libs += [
   pcollections: "org.pcollections:pcollections:$versions.pcollections",
   opentelemetryProto: "io.opentelemetry.proto:opentelemetry-proto:$versions.opentelemetryProto",
   reflections: "org.reflections:reflections:$versions.reflections",
+  slf4jReload4j: "org.slf4j:slf4j-reload4j:$versions.slf4j",
```

Comment:
Could you please move it down to around `slf4jApi`?

Reply:
After renaming, missed to shuffle. Thanks. Updated.